src/scrappers/source/proCyclingStats/raceStageResults.js

```
@@ -200,361 +259,345 @@ function climb(label) {
-  } catch (exception) {
-    logError(
-      "Race Stage Results",
-      `Exception in scrapeStage waiting for  selector '.page-content' on '${url}'`,
-      exception,
-    );
-    return null;
-  }
+  // Handle time cells with hidden spans
+  if (cell.classList.contains("time")) {
+    // Always remove hidden spans first
+    cell.querySelectorAll("span.hide").forEach((span) => span.remove());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)